### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by jlpedrosa**
Inital commit:
* Chart created
  - Deployment
  - Volumes for config and media (static or dynamically provisioned)
  - Ingress

* CI
  - A workflow to be invoked from jellyfin main repo when a new release is created, updates versions
  - A workflow to release the helm release

* Release
  - Helm releases uses the GH pages, on the first chart it will create the `gh-branch` so it can be pullled and installed

* Pending:
  - PR to update jellyfin.org with the documentation
  - Iron out possible issues on first release.
  
  I can't asign reviewers.. so.. cc: @crobibero, @joshuaboniface
  
It works in my cluster ;)   
image

### CODE REVIEW

File: charts/jellyfin/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+appVersion: "10.9.0"
```
Comment (jlpedrosa) [inline charts/jellyfin/Chart.yaml:2]:
```suggestion
appVersion: "10.9.1"
```

File: charts/jellyfin/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+appVersion: "10.9.1"
+description: Jellyfin Media Server
+name: jellyfin
+version: 10.9.0
```
Comment (jlpedrosa) [inline charts/jellyfin/Chart.yaml:5]:
```suggestion
version: 10.9.1
```

File: charts/jellyfin/values.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
+# Default values for jellyfin.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: docker.io/jellyfin/jellyfin
+  tag: "10.9.0"
```
Comment (jlpedrosa) [inline charts/jellyfin/values.yaml:9]:
```suggestion
  tag: ""
```

File: .github/renovate.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json"
+}
```
Comment (nielsvanvelzen) [inline .github/renovate.json:3]: Should use the organisation renovate config:
https://github.com/jellyfin/.github/blob/master/renovate-presets/default.json

e.g. https://github.com/jellyfin/jellyfin/blob/master/.github/renovate.json but with the default preset

**Comment by Isonami** [comment]
May I have a suggestion here, or better to create new issue?

It might be helpful to set pod labels/annotations trough values in some situations (metrics, policies, allocation rules and etc).

File: .github/renovate.json (hunk 4 above)
Comment (jlpedrosa) [reply .github/renovate.json:3]: The default config works fine for helm charts, at least i's working fine for my other repos. If you want I can delete this file and let renovate onboard this repo?

**Comment by jlpedrosa** [comment]
> May I have a suggestion here, or better to create new issue?
> 
> It might be helpful to set pod labels/annotations through values in some situations (metrics, policies, allocation rules and etc).

Defo we need to add the labels and annotations. said that, I'd probably suggest put it in another PR, so we can get the CI working here and documentation.

File: .github/renovate.json (hunk 4 above)
Comment (jlpedrosa) [reply .github/renovate.json:3]: Renovate already kicked in: https://github.com/jellyfin/jellyfin-helm/pull/1/ 

Do you want to merge 1 and then I rebase this one?

Comment (nielsvanvelzen) [reply .github/renovate.json:3]: You can just add the organization default preset in this PR and we'll close the automated Renovate PR.

Comment (jlpedrosa) [reply .github/renovate.json]: done.

Comment (nielsvanvelzen) [reply .github/renovate.json]: That isn't our organization default preset

File: .github/renovate.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ]
```
Comment (jlpedrosa) [inline .github/renovate.json:5]:
```suggestion
  "extends": [
    "github>jellyfin/.github//renovate-presets/default"
  ]
```

File: .github/renovate.json (hunk 4 above)
Comment (jlpedrosa) [reply .github/renovate.json]: I updated it for the organization default, not sure if we should (in next prs) create a new one for helm...

**Comment by jlpedrosa** [comment]
@crobibero Are we good to go?

**Comment by crobibero** [comment]
Still needs a license before merging

**Comment by jlpedrosa** [comment]
> Still needs a license before merging

@crobibero 
https://github.com/jellyfin/jellyfin-helm/pull/3

**Review by crobibero — APPROVED** [review]

**Pull request merged**